docs/references/go/other-examples.mdx

```diff
@@ -5,30 +5,72 @@ description: Explore different examples of how to utilize Clerk with Go.
 
 # Examples
 
-Below is a code snippet showing what you can do from your backend with Clerk. Look at the [API reference](/docs/references/backend/overview) for a full list of requests you can make from your backend.
+Let's see an example that calls a few different [Clerk Backend API](/docs/references/backend/overview) operations using the Clerk Go SDK.
 
-## Update metadata
+In the snippet below, you will create an organization and update its slug. You'll fetch all organization
+memberships, loop through them to get the first one, and get more details about the organization's user.
 
-Add a new key/value to the private metadata stored on a user.
+Your Clerk secret key is required. If you are signed into your Clerk Dashboard, your secret key should become visible by clicking on the eye icon. Otherwise, you can retrieve your Clerk secret key from the Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.
 
+<InjectKeys>
 ```go
-var client clerk.Client
-
-func addStripeCustomerID(user *clerk.User, stripeCustomerID string) error {	
-	currPrivMetadata := user.PrivateMetadata.(map[string]interface{})	
-	currPrivMetadata["stripe_customer_id"] = stripeCustomerID
-
-	var keyValStrings []string
-	for key, value := range currPrivMetadata {
-		keyValStrings = append(keysString, fmt.Sprintf("\"%s\": \"%s\"", key, value))
-	}
-	finalStr := fmt.Sprintf("{%s}", strings.Join(keysString, ","))
-	user, err := s.clerkClient.Users().Update(sess.UserID, &clerk.UpdateUser{
-		PrivateMetadata: finalStr,
-	})
-
-	if err != nil {
-		panic(err)
-	}
+import (
+    "github.com/clerk/clerk-sdk-go/v2"
+    "github.com/clerk/clerk-sdk-go/v2/organization"
+    "github.com/clerk/clerk-sdk-go/v2/organizationmembership"
+    "github.com/clerk/clerk-sdk-go/v2/user"
+)
+
+func main() {
+  // Each API operation requires a context.Context as the first argument.
+  ctx := context.Background()
+
+  // Set the API key
+  clerk.SetKey(`{{secret}}`)
+
+  // Create an organization
+  org, err := organization.Create(ctx, &organization.CreateParams{
+      Name: clerk.String("Clerk Inc"),
+  })
+  if err != nil {
+    // You can get additional information on the error, if it can
+    // be type-cast to clerk.APIErrorResponse.
+    if apiErr, ok := err.(*clerk.APIErrorResponse); ok {
+      apiErr.TraceID
+      apiErr.Error()
+      apiErr.Response.RawJSON
+    }
+    // handle the error
```

Comment:
```suggestion
    // handle the error
    panic(err)
```

Does it make sense to add `panic` here as well?

Reply:
Sure, added!